book/src/call_rust_from_cpp/special_types.md

```diff
@@ -17,3 +17,42 @@ tries to support them using C++ feature called User-defined Literals.
 | `'a'_rs_b`     | `b'a'`          | `uint8_t`                         | Not Implemented | unconditionally     |
 | `"hello"_rs_b` | `b"hello"`      | `rust::Ref<rust::Slice<uint8_t>>` | Not Implemented | `[u8]`              |
 | `"hello"_rs_c` | `c"hello"`      | `rust::Ref<rust::ffi::CStr>`      | Not Implemented | `::rust::ffi::CStr` |
+
+## Fields as underlying types
+
+When you declare fields in a tuple or struct using `field name (offset = X, type = T);`, the generated C++ exposes helper wrapper types:
```

Comment:
I would suggest a new section for fields. Fields are not (special) types in the way that `Bool`, `Char`, and `Ref<T>` are types.